src/base64.h

```diff
@@ -48,8 +49,9 @@ size_t base64_decoded_size(const TypeName* src, size_t size) {
 extern const int8_t unbase64_table[256];
 
 
-#define unbase64(x)                                                           \
-  static_cast<uint8_t>(unbase64_table[static_cast<uint8_t>(x)])
+inline static int8_t unbase64(uint8_t x) {
```

Comment:
If IIUC if you declare `unbase64_table` as `constexpr` you could declare this function as `constexpr` as well.